src/Sylius/Behat/Page/Admin/ProductAttribute/UpdatePage.php

```diff
@@ -62,7 +63,13 @@ public function changeAttributeValue(string $oldValue, string $newValue): void
      */
     public function hasAttributeValue(string $value): bool
     {
-        return null !== $this->getElement('attribute_choice_list_element', ['%value%' => $value]);
+        try {
+            $attributeValue = $this->getElement('attribute_choice_list_element', ['%value%' => $value]);
+        } catch (ElementNotFoundException $exception) {
+            return false;
+        }
+
+        return null !== $attributeValue;
```

Comment:
I think it's not possible for `getElement(...)` to return `null` value - so we could just type `return true;` here.